### ISSUE TITLE
SparseGaussianAdam import error

### ISSUE THREAD

**Issue by luckyyf0517** [wenchaozheng/WRF-GSplus#4]
Hello, thank you very much for your open-source work.

While trying to reproduce the results from the paper, I noticed that the SparseGaussianAdam is not implemented in the diff_gaussian_rasterization library under the submodule folder. This means that SPARSE_ADAM_AVAILABLE is never enabled in the training code.

I would like to ask if this SPARSE_ADAM optimizer affects the final metrics in the experiments. Using the code from this repository, the Median SSIM metric I'm currently able to reproduce is 0.88, which hasn't reached the 0.90 reported in the paper.

**Comment by wenchaozheng** [comment]
Yes, this part was not used in our process because the reference to the original 3DGS structure was not well-organized, but it is not the reason that affected the results. 
In fact, 0.90 is to maintain consistency with the significant digits of the previous paper, and the actual value is rounded from 0.8957 (as shown in the screenshot).
At the same time, in terms of rendering spectra, it is entirely possible to render single channels and calculate the loss of GT to obtain higher SSIM. We separate the real and imaginary parts of the two channels to increase interpretability and unify them with the format of the CSI case study.

<img width="391" height="458" alt="Image" src="https://github.com/user-attachments/assets/9912cfac-ff45-4898-a409-a2fd4e1b135a" />
